Formula/llvm.rb

```diff
@@ -95,66 +125,95 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    # Polly is --HEAD-only because it requires isl and the version of Polly
-    # shipped with 3.6.2 only compiles with isl 0.14 and earlier (current
-    # version is 0.15). isl is distributed with the Polly source code from LLVM
-    # 3.7 and up, so --HEAD builds do not need to depend on homebrew isl.
-    option "with-polly", "Build with the experimental Polly optimizer"
     resource "polly" do
       url "http://llvm.org/git/polly.git"
     end
   end
 
-  keg_only :provided_by_osx
+  # keg_only :provided_by_osx if OS.mac?
 
-  option :universal
-  option "with-clang", "Build the Clang compiler and support libraries"
+  # Options
+  option :universal if OS.mac?
```

Comment:
> Are `OS.mac?` and `OS.linux?` for Linuxbrew only?

Exactly. The formulae in this repository exclusively target OS X, so we don't need to distinguish between OS X and Linux in them.